assessments/0004-tremor.md

```diff
@@ -67,9 +68,9 @@ Scale:
     - The [scripts](https://github.com/tremor-rs/tremor-www-docs/tree/main/python_scripts) which generate certain pieces of documentation introduce fragility into the system: if the location of files, names of repos, etc. change then the documentation pipeline breaks. This also makes it hard for non-"technical" users to make changes to these pages, and to reorganize these pages if needed.
         - The automation benefits might be worth it however. 
     - Your [release process](https://github.com/tremor-rs/tremor-www-docs/blob/main/RELEASE_PROCESS.md) is documented, which is great to see!
-    - Your site is only partially serachable, with sections not within /docs inaccessible by search.
+    - Your site is only partially searchable, with sections not within /docs inaccessible by search.
     
-**Content creation processes:** There are no content creation processes documented and no CONTRIBUTING.md file for the main docs repository. Some things to think about:
+- **Content creation processes:** There are no content creation processes documented and no CONTRIBUTING.md file for the main docs repository. Some things to think about:
```

Comment:
Just curious @nate-double-u: what changed in this line? Although they're not, they _look_ the same in the online diff. Was it a change from a tab to a space or a hyphen to a dash, or some other hidden characters?

Reply:
I've added a `-` at the beginning of the line for consistency.

Reply:
(also, adding the `-` in  some cases is what triggers the renderer to treat the next paragraph as a bulleted list. I thought this was the cleanest way to sort it -- we may want to revisit the template and use section H3 or H4 section headers instead in situations like this)

Reply:
(Oh, of course! Lol. I read the diff too quickly and took the diff `-` as a markdown list character! 🤷‍♂️ 
Right, it makes sense for consistency to include it in the list.

Reply:
One `-` added at the beginning of the line is a bit of a sneaky diff...